src/store/authors/reducer.js

```diff
@@ -0,0 +1,39 @@
+import {
+	ADD_COURSE_AUTHOR,
+	CREATE_AUTHOR,
+	REMOVE_COURSE_AUTHOR,
+	RESET_COURSE_AUTHORS,
+	SAVE_AUTHORS,
+} from './actionTypes';
+
+const authorsInitialState = { authors: [], courseAuthors: [] };
+
+export default function authorsReducer(
+	state = authorsInitialState,
+	{ type, payload }
+) {
+	switch (type) {
+		case SAVE_AUTHORS: {
+			return { ...state, authors: payload };
+		}
+		case CREATE_AUTHOR: {
```

Comment:
CREATE_AUTHORS is basically SAVE_AUTHOR. See the comment above, description from hometask

Reply:
Changed